From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ariel D'Alessandro <ariel.dalessandro@collabora.com>
Date: Tue, 17 Dec 2024 15:55:14 -0300
Subject: GENIO: configs: genio-350-evk: Enable CONFIG_WDT_MTK

Enable MediaTek watchdog timer as it comes enabled from BL2 TFA.
Otherwise, this causes a reset after a certain amount of time.

Signed-off-by: Ariel D'Alessandro <ariel.dalessandro@collabora.com>
---
 configs/genio_350_evk_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/genio_350_evk_defconfig b/configs/genio_350_evk_defconfig
index 111111111111..222222222222 100644
--- a/configs/genio_350_evk_defconfig
+++ b/configs/genio_350_evk_defconfig
@@ -110,6 +110,7 @@ CONFIG_USB_GADGET_VENDOR_NUM=0x0e8d
 CONFIG_USB_GADGET_PRODUCT_NUM=0x201c
 CONFIG_USB_ETHER=y
 CONFIG_WDT=y
+CONFIG_WDT_MTK=y
 CONFIG_OF_LIBFDT_OVERLAY=y
 CONFIG_EFI_SET_TIME=y
 CONFIG_EFI_RUNTIME_UPDATE_CAPSULE=y
-- 
Armbian

